### PR TITLE
Update shaping.toml

### PR DESCRIPTION
**Pull request by Solmea**
* shaping rules for gmail.com
* shaping rules for outlook.com
* shaping rules for yahoo.com
* shaping rules for web.de
* shaping rules for orange.fr
* shaping rules for qq.com
* shaping rules for 163.com

Tested on a production server running  2024.06.10-84e84b89

### CODE REVIEW

**Comment by MHillyer** [comment]
We expect the default shaping to be used in combination with the community one, so there is no need to reproduce the default automation, can you remove it?

**Comment by Solmea** [comment]
I think I removed all defaults . Also removed the outlook.com default as that is also in the policy-extra shaping toml

File: assets/community/shaping.toml [hunk 1]
```diff
@@ -26,4 +26,264 @@
...
+[["outlook.com".automation]]
+regex = "Temporary server error. Please try again later"
+trigger = {Threshold="5/hr"}
+action = "Suspend"
+duration = "1 hour"
+
+#===== orange.fr
+["orange.fr"]
+max_deliveries_per_connection = 75
+connection_limit = 1
```
Comment (Solvik) [inline assets/community/shaping.toml]:
```suggestion
connection_limit = 2
```

The postmaster page says: `Sending limits: Orange currently allows up to two simultaneous connections per sending IP address, and up to 100 recipients per message`

Comment (Solmea) [reply assets/community/shaping.toml]: Thanks, good call. I did not check the postmastery page again.. But usually in my experience orange was very strict on sending too much/fast.

**Review by MHillyer — APPROVED** [review]

**Pull request merged**